one-pager.md

```diff
@@ -1,33 +1,34 @@
 # Pioneer Valley Housing Co-op Seeks Property 
 
-Are you looking to do something unique with your land that fosters community, equity, sustainability, and stewardship for generations to come? We are a group of renters who live in the Pioneer Valley who have joined together as a group equity housing co-op. We are creating a local alternative to traditional homeownership where we can collectively invest in ourselves, our community, and the environment. 
+Are you interested in housing that fosters community, equity, sustainability, and stewardship for generations to come? What about a local alternative to renting and traditional homeownership? We are forming a group equity housing co-op to collectively invest in ourselves, our community, and the environment.
 
-We are looking for access to land either by donation, agreement or sale with the possibility of the landowner continuing occupancy. We would love to connect with like-minded people and organizations with the following values:
-
-**Community**: We believe that resident wellbeing and community wellbeing is interdependent.
```

Comment:
this isn't really a belief statement - could be stronger by expressing the connection between supporting individuals wellbeing (creative projects, financially, flexible commitments, stability), residential community (rich social life) and broader local community (bumpin)

Reply:
"we believe in a pleasent place to live"